test/cases/railtie_test.rb

```diff
@@ -57,19 +57,11 @@ def setup
     assert_nil SignedGlobalID.expires_in
   end
 
-
-  test 'SignedGlobalID.verifier defaults to Blog::Application.message_verifier(:signed_global_ids) when secret_key_base is present' do
```

Comment:
I'm no longer sure this should have been removed.  Checking now.

Reply:
Shouldn't really be removed, but it does need to be rewritten, since `Rails::Application#message_verifier` isn't available until 4.1.

It should be something like
```
generated_key = OpenSSL::PKCS5.pbkdf2_hmac_sha1(app.config.secret_token, 'signed_global_ids', 1000, 64)
assert_equal ActiveSupport::MessageVerifier.new(generated_key).generate(message), signed_message
```